e2e-tests/insurance/cypress/e2e/journeys/account/password-reset/account-password-reset-retry/account-password-reset-retry-maximum-retries.spec.js

```diff
@@ -9,70 +7,73 @@ const {
-  const baseUrl = Cypress.config('baseUrl');
-  const passwordResetUrl = `${baseUrl}${PASSWORD_RESET_ROOT}`;
-  const accountSuspendedUrl = `${baseUrl}${SUSPENDED_ROOT}`;
+context(
+  'Insurance - Account - Password reset - Submitting the form successfully over the maximum threshold in the allowed time period should block the account',
+  () => {
+    const baseUrl = Cypress.config('baseUrl');
+    const passwordResetUrl = `${baseUrl}${PASSWORD_RESET_ROOT}`;
+    const accountSuspendedUrl = `${baseUrl}${SUSPENDED_ROOT}`;
 
-  let account;
+    let account;
 
-  before(() => {
-    cy.deleteAccount();
+    before(() => {
+      cy.deleteAccount();
 
-    cy.completeAndSubmitCreateAccountForm({ navigateToAccountCreationPage: true });
+      cy.completeAndSubmitCreateAccountForm({ navigateToAccountCreationPage: true });
 
-    // go back to create account page
-    cy.clickBackLink();
+      // go back to create account page
+      cy.clickBackLink();
 
-    // navigate to sign in page
-    yourDetailsPage.signInButtonLink().click();
+      // navigate to sign in page
+      cy.clickSignInButtonLink();
 
-    // navigate to password reset page
-    signInPage.resetPasswordLink().click();
+      // navigate to password reset page
+      cy.clickSignInResetPasswordLink();
 
-    cy.assertUrl(passwordResetUrl);
-  });
+      cy.assertUrl(passwordResetUrl);
+    });
 
-  describe('when attempting password reset multiple times and reaching the maximum retries threshold', () => {
-    beforeEach(() => {
-      cy.saveSession();
+    describe('when attempting password reset multiple times and reaching the maximum retries threshold', () => {
+      beforeEach(() => {
+        cy.saveSession();
 
-      cy.navigateToUrl(passwordResetUrl);
+        cy.navigateToUrl(passwordResetUrl);
 
-      cy.completeAndSubmitPasswordResetFormMaximumRetries();
-    });
+        cy.completeAndSubmitPasswordResetFormMaximumRetries();
+      });
 
-    it(`should redirect to ${SUSPENDED_ROOT} with ID query param`, () => {
-      /**
-       * Get the account ID directly from the API,
-       * so that we can assert that the URL has the correct account ID.
-       */
-      const accountEmail = Cypress.env('GOV_NOTIFY_EMAIL_RECIPIENT_1');
+      it(`should redirect to ${SUSPENDED_ROOT} with ID query param`, () => {
+        /**
+         * Get the account ID directly from the API,
+         * so that we can assert that the URL has the correct account ID.
+         */
+        const accountEmail = Cypress.env('GOV_NOTIFY_EMAIL_RECIPIENT_1');
 
-      cy.getAccountByEmail(accountEmail).then((responseData) => {
-        const [firstAccount] = responseData;
-        account = firstAccount;
+        cy.getAccountByEmail(accountEmail).then((responseData) => {
+          const [firstAccount] = responseData;
+          account = firstAccount;
```

Comment:
could we have a let accountUrl and construct it here then we don't need to construct it twice?

Reply:
This wasn't changed in this PR, this is just eslint/prettier, but i've updated this so there is an `expectedUrl` const.